app/models/api/v3/qual.rb

```diff
@@ -31,6 +33,13 @@ def simple_type
         'qual'
       end
 
+      def download_original_attribute(context)
+        Api::V3::DownloadQual.
+          joins(:download_attribute).
+          where('download_attributes.context_id' => context.id, qual_id: id).
```

Comment:
maybe `find_by`? I know it works the same but it's more semantical if there's only one element like that